canonical-websites/www.ubuntu.com
# 2580: Build resource page enhancements

## Done

- Updated resources page, which includes:
  - New header strip
  - New styling for the articles cards
  - Removal of search box (for now)
  - Removal of tab label in lieu of an 'All Topics' tab

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/resources
- Check that the styling is in line with the [new design](https://github.com/canonical-websites/www.ubuntu.com/issues/2565)
- Check that filtering by Topic works as expected
- Check that filtering by Content Type works as expected
- Check that everything looks ok for different screen sizes and browsers (as per [document](https://github.com/canonical-webteam/practices/blob/master/coding/browser-support.md))


## Issue / Card

Fixes #2580 

## Screenshots
New design:

![image](https://user-images.githubusercontent.com/9105999/34950541-5dc6aaa6-fa0b-11e7-81cf-c2177967a580.png)
